resource_booking/models/resource_resource.py

```diff
@@ -14,3 +16,17 @@ def _check_bookings_scheduling(self):
             [("combination_id.resource_ids", "in", self.ids)]
         )
         return bookings._check_scheduling()
+
+    def is_available(self, start_dt, end_dt, domain=None, tz=None):
+        """Convenience method to check whether a resource is available within a
+        time span.
+        """
+        self.ensure_one()
+
```

Comment:
Please avoid empty lines inside a method that makes it less readable. The comment lines already act as separator

Reply:
done